portfolio/src/main/webapp/script.js

```diff
@@ -105,9 +117,9 @@ function buildInfoWindow(lat,lng){
     return clone;
 }
 
+/** Manages the data of the report once the info window pops up. */
 function submitFormData(element){
-    postMarker(markerLat, markerLng, getRadioValueCrimes(element), document.getElementById('date').value, document.getElementById('time').value, document.getElementById('address').value, document.getElementById('description').value);
-    createMarkerForDisplay(markerLat, markerLng, getRadioValueCrimes(), document.getElementById('date').value, document.getElementById('time').value, document.getElementById('address').value, document.getElementById('description').value);
+    postMarker(markerLat, markerLng, getRadioValueCrimes(), document.getElementById('date').value, document.getElementById('time').value, document.getElementById('address').value, document.getElementById('description').value);
```

Comment:
this tabbing is very difficult to read. Use:
postMarker(
  markerLat,
  markerLng,
  ...);

Reply:
Sounds good, thank you!